packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/job-logs.html

```diff
@@ -31,11 +31,5 @@
                 ></span
             ></oui-button>
         </div>
-        <div
-            class="col-md-12 logger_placeholder"
-            data-ng-if="$ctrl.isJobTerminated()"
-        >
-            {{ ::'data_processing_details_logs_terminated' | translate }}
```

Comment:
Following translation still needed?
see: https://github.com/ovh/manager/blob/develop/packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-logs/translations/Messages_fr_FR.json#L5

Reply:
Thanks for the review. I removed the translation that was not used anymore